frontend/package.json

```diff
@@ -14,7 +15,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest",
```

Comment:
we should use the react-scripts test not just strictly jest there is some magic it does for us